runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -390,8 +406,32 @@ def _start_ray_workers(self, ray_port, env_vars):
 
         time.sleep(5)
 
+    def _update_cluster_status_from_sky_status(self, sky_status: str):
+        if sky_status == "UP":
```

Comment:
one thought to explicitly use sky values here, we could do something like:
```
from sky import ClusterStatus as SkyClusterStatus
if sky_status == SkyClusterStatus.UP
```

Reply:
Good call